core/src/epicli/data/common/ansible/playbooks/roles/prometheus/tasks/configure-k8s-apps-monitoring.yml

```diff
@@ -22,16 +22,16 @@
 - name: apply rolebinding to k8s for prometheus
   become: yes
   become_user: "{{ admin_user.name }}"
-  shell: "kubectl apply --kubeconfig=/home/{{ admin_user.name }}/.kube/config -f /home/{{ admin_user.name }}/k8s-rolebinding.yml"
+  shell: "kubectl apply -f /home/{{ admin_user.name }}/k8s-rolebinding.yml"
   run_once: true
-  delegate_to: "{{ master_hostname }}"
+  delegate_to: localhost
 
 - name: Get kubernetes bearer token for prometheus
   become: yes
   become_user: "{{ admin_user.name }}"
-  shell: "kubectl --kubeconfig=/home/{{ admin_user.name }}/.kube/config describe secret $(kubectl get secrets --namespace=kube-system | grep prometheus | awk '{print $1}') --namespace=kube-system | grep -E '^token' | awk '{print $2}' | head -1"
+  shell: "kubectl describe secret $(kubectl get secrets --namespace=kube-system | grep prometheus | awk '{print $1}') --namespace=kube-system | grep -E '^token' | awk '{print $2}' | head -1"
```

Comment:
I don't like this line, too much bash magic I think we should refactor it. 🤔

Reply:
How about this one?

kubectl get $(kubectl get secrets -n kube-system -o name | grep prometheus) -n kube-system -o jsonpath='{ .data.token }'

Reply:
Haven't tested it, but looks much better! :+1:

Reply:
![image](https://user-images.githubusercontent.com/6271862/93345050-aea00f80-f832-11ea-8e98-4e51470e4c88.png)